doc/06-helper.md

```diff
@@ -237,9 +237,9 @@ You can find more about how `.env` file loading and overloading works on
 
 ## Other helpers
 
-* `get_context()` returns the initial `Context`. See the [context
+* `ctx()` returns the initial `Context`. See the [context
   documentation](./05-context.md) for mor information
 * `variable()` returns a variable stored in the  `Context`. See the [context
   documentation](./05-context.md) for mor information
-* `get_loger()` returns the current `Logger`. See the [logger
+* `get_logger()` returns the current `Logger`. See the [logger
```

Comment:
Good catch! Why don't you rename it to `logger()`?

Reply:
i'm not sure, we want to avoid having the same name as the return class, i was able to do that for everything expect this one and the cache (but i have another idea for the cache)